bots/csr/src/main/java/org/openjdk/skara/bots/csr/CSRBot.java

```diff
@@ -70,10 +67,6 @@ private String describe(PullRequest pr) {
         var prs = repo.pullRequests();
 
         for (var pr : prs) {
-            if (!cache.needsUpdate(pr)) {
-                continue;
-            }
```

Comment:
An alternative to eliminating this check would be to poll JBS either if the PR was updated or if the PR has a csr label. But as you say, maybe it won't be a performance problem, in which case your proposed change is better.

Reply:
Note that we also want to react if a CSR issue is created when the label isn't there. In that case we automatically add the CSR label.

Reply:
Yeah, that's the main reason your fix is better.